### ISSUE TITLE
Issue: File delete not working ?

### ISSUE THREAD

**Issue by nyctonio** [Nushcode/CollectWiz#1]
The issue on the latest release file not getting deleted 
As I can see you used React as your tech stack and your filesystem is based on the routes which left a lot of bugs in the project 
as you can create any random folder or file on any route as there is no such validation on your route handling and the file gets created with no link to the parent folder a better approach would be to use some better data structure like b-tree 
have a look at this article [Read This ✌️](https://softwareengineering.stackexchange.com/questions/289395/efficient-data-structure-to-implement-fake-file-system
@CoderChirag @Nushcode ek baar dekh lena yaar 🌝

**Comment by CoderChirag** [comment]
Yeah @nyctonio we know the issue as we didn't have time to think design properly before starting due to a very short span of time for making this. But we are planning to fix it in future.
And thanks for your article recommendation buddy ✌🏻, we will surely be looking at it.
